packages/core/src/nuxt.js

```diff
@@ -26,7 +26,8 @@ export default class Nuxt extends Hookable {
 
     // Deprecated hooks
```

Comment:
Not related to that PR but while you are in here

A `TODO` to remove the hooks in Nuxt 3 would be :+1: 
Do we have warnings for them in place (when using the deprecated ones)?

Reply:
Yeah. Hookable has built-in support for deprecated hooks and throws warn.

Reply:
Will add TODO